js_modules/dagit/packages/core/src/scripts/generateGraphQLTypes.ts

```diff
@@ -45,6 +42,18 @@ execSync(
   {stdio: 'inherit'},
 );
 
+// Write schema.graphql in the SDL format, with descriptions
+const resultWithDescriptions = execSync(
+  `dagster-graphql --ephemeral-instance --empty-workspace -t '${getIntrospectionQuery()}'`,
+  {cwd: '../../../../examples/docs_snippets/'},
+).toString();
+const schemaJsonWithDescriptions = JSON.parse(resultWithDescriptions).data;
+const sdlWithDescriptions = printSchema(buildClientSchema(schemaJsonWithDescriptions));
+
+console.log('Generating schema.graphql with descriptions...');
+
+writeFileSync(TARGET_FILE, sdlWithDescriptions);
```

Comment:
The `schema.graphql` is initially generated without descriptions. Otherwise, the generated types in typescripts would also have descriptions.

Reply:
this might be worth including as a comment in the  code too